platform/android/MapboxGLAndroidSDKTestApp/src/main/res/layout/activity_marker_view.xml

```diff
@@ -18,6 +18,7 @@
         android:layout_below="@id/toolbar"
         app:center_latitude="38.907192"
         app:center_longitude="-77.036871"
+        app:texture_mode="true"
```

Comment:
@tobrun Is it intended to leave this activity and `PolygonActivity` with a `TextureView` by default? If so, we need to clearly say it. A better way might be to a) have a specific `TextureViewActivity` example or b) leave the `SurfaceView` as the default in both activities, but add a clear comment that with that line you'd enable the now deprecated `TextureView`.